sections/introduction.tex

```diff
@@ -10,9 +10,18 @@ \section{Introduction}
 
 Since then, we have noticed rapid development in this area. A crucial milestone was
 the creation of AlphaGo Zero which, in contrast to its predecessors, used only reinforcement
-learning during training and yet managed to outperform all the older versions in only forty
-days \cite{AlphaGoZeroBlog}. That opened up opportunities for AI to master other games like chess 
-or shogi.
+learning during training and yet managed to outperform all of the older versions in only forty
```

Comment:
Seems like you accidentally reverted the change introduced by #22, probably during conflict resolution:
```suggestion
learning during training and yet managed to outperform all the older versions in only forty
```